File: Lesson 3/ClassExamples/bank.py
class BankAccount:
    def __init__(self, account_number, balance=0):
        self.account_number = account_number
        self.balance = balance

    def deposit(self, amount):
        self.balance += amount
        print(f"Deposited ${amount}. New balance: ${self.balance}")

    def withdraw(self, amount):
        if self.balance >= amount:
            self.balance -= amount
            print(f"Withdrew ${amount}. New balance: ${self.balance}")
        else:
            print("Insufficient funds!")


# Example usage
account = BankAccount("12345")
account.deposit(1000)  # Output: Deposited $1000. New balance: $1000
account.withdraw(500)  # Output: Withdrew $500. New balance: $500
account.withdraw(1000)  # Output: Insufficient funds!

File: Lesson 6/Bank.java
public class Bank {
    // Attributes
    private String accountNumber;
    private double balance;
    private String ownerName;

    // Setters
    public void setAccountNumber(String accountNumber) {
        this.accountNumber = accountNumber;
    }

    public void setBalance(double balance) {
        this.balance = balance;
    }

    public void setOwnerName(String ownerName) {
        this.ownerName = ownerName;
    }

    // Getters
    public String getAccountNumber() {
        return accountNumber;
    }

    public double getBalance() {
        return balance;
    }

    public String getOwnerName() {
        return ownerName;
    }

    // Example usage
    public static void main(String[] args) {
        // Creating a Bank object
        Bank bankAccount = new Bank();

        // Setting attributes using setter methods
        bankAccount.setAccountNumber("123456789");
        bankAccount.setBalance(1000.0);
        bankAccount.setOwnerName("Alice");

        // Getting and displaying attributes using getter methods
        String accountNumber = bankAccount.getAccountNumber();
        double balance = bankAccount.getBalance();
        String ownerName = bankAccount.getOwnerName();

        System.out.println("Account Number: " + accountNumber);
        System.out.println("Balance: $" + balance);
        System.out.println("Owner Name: " + ownerName);
    }
}
